File: app/src/main/java/com/skf/bingeflix/Adapters/MovieItemClickListener.java
package com.skf.bingeflix.Adapters;

import android.widget.ImageView;

import com.skf.bingeflix.Models.Movie;

public interface MovieItemClickListener {

    void onMovieClick(Movie movie, ImageView movieImageView); // we will need the imageview to make the shared animation between the two activity

}

File: app/src/main/java/com/skf/bingeflix/Adapters/MovieItemClickListener.kt
package com.skf.bingeflix.Adapters

import android.widget.ImageView
import com.skf.bingeflix.Models.Movie

interface MovieItemClickListener {
    fun onMovieClick(movie: Movie?, movieImageView: ImageView?) // we will need the imageview to make the shared animation between the two activity
}